test/specs/public-api.spec.js

```diff
@@ -15,6 +15,7 @@ describe('Riot public api', () => {
       'component',
       'pure',
       'withTypes',
+      'composeTypes',
```

Comment:
I think that this method shouldn't belong to the default RIot.js api and can be easily implemented outside of the framework